sdk/paloaltonetworks/Azure.ResourceManager.PaloAltoNetworks/src/autorest.md

```diff
@@ -0,0 +1,49 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+azure-arm: true
+csharp: true
+library-name: PaloAltoNetworks
+namespace: Azure.ResourceManager.PaloAltoNetworks
```

Comment:
Has this namespace been reviewed?

Reply:
Namespace review is complete now - https://github.com/Azure/azure-sdk-pr/issues/892
The approved namespace is for .NET is "Azure.ResourceManager.PaloAltoNetworks.Ngfw".
I have regererated the code to reflect this change.